client/src/js/components/bhLatestInvoice.js

```diff
@@ -27,8 +27,11 @@ function LatestInvoice(Patient, moment, Notify, Session) {
   /** getting patient document */
   function startup() {
     if (!vm.debtorUuid) { return; }
+    vm.loading = true;
+
     Patient.latest(vm.debtorUuid)
       .then(function (patientInvoice) {
+        vm.loading = false;
```

Comment:
It would be better to put the `vm.loading = false` call in the `.finally()` method of the promise.  That way, even if the query errors, the loading indicator will be removed.